pallets/validators-manager/src/lib.rs

```diff
@@ -257,8 +256,7 @@ pub mod pallet {
         ) -> DispatchResultWithPostInfo {
             ensure_root(origin)?;
             let validator_account_ids =
-                Self::validator_account_ids().or_else(|| Some(vec![])).expect("empty vec");
-            ensure!(validator_account_ids.len() > 0, Error::<T>::NoValidators);
+                Self::validator_account_ids().ok_or(Error::<T>::NoValidators)?;
```

Comment:
why did we remove the explicit check that the length is > 0? What exactly does `ok_or` do in this case?